monitor/linuxmonitor/processor.go

```diff
@@ -46,47 +53,31 @@ func NewLinuxProcessor(collector collector.EventCollector, puHandler monitor.Pro
 
 // Create handles create events
 func (s *LinuxProcessor) Create(eventInfo *rpcmonitor.EventInfo) error {
-	contextID, err := generateContextID(eventInfo)
-	if err != nil {
-		return fmt.Errorf("Couldn't generate a contextID: %s", err)
+
+	if !s.regStart.Match([]byte(eventInfo.PUID)) {
+		return fmt.Errorf("Invalid PU ID")
 	}
 
-	return s.puHandler.HandlePUEvent(contextID, monitor.EventCreate)
+	return s.puHandler.HandlePUEvent(eventInfo.PUID, monitor.EventCreate)
 }
 
 // Start handles start events
 func (s *LinuxProcessor) Start(eventInfo *rpcmonitor.EventInfo) error {
 
-	contextID, err := generateContextID(eventInfo)
-	if err != nil {
-		return err
+	// Validate the PUID format
+	if !s.regStart.Match([]byte(eventInfo.PUID)) {
+		return fmt.Errorf("Invalid PU ID")
```

Comment:
can you provide the thing that was invalid in the returned error?

Reply:
ok